edudoc/guidelines-for-disciplinary-incidents/guidelines-for-disciplinary-incidents.md

```diff
@@ -1,93 +1,91 @@
-  * Example: a young competitor doesn’t start their timer correctly and becomes upset as their attempt has been given a DNF penalty. Speak to their parents, use more straightforward terms such as “disqualify” rather than DNF, to explain what happened. Show the competitor how to start the timer correctly. Tell them about a time it happened to you, maybe a time when you had a particularly good scramble. Give them tips on how you have avoided this happening again.
-* If you are becoming frustrated during an interaction, ask for help early or get another Delegate/organizer/trusted person to step in. If this is not possible consider taking a break from the situation and rediscussing the matter with the people involved at a later point in the competition.
-* When there are multiple difficult situations occurring at competitions, try and prioritize what is the most important and where possible assign others (either Delegates, organizers or other trusted individuals) to deal with the less important or less time-critical issues.
-* After you have managed a difficult situation, it is important to debrief. Speak with the other Delegates/organizers/trusted people involved; discuss what went well, what could’ve been improved, what you learned, and avoid blaming others if things went poorly. If you were the only person involved, reach out to a Delegate you trust and debrief.
-* Use the Delegate report as a resource to gather feedback on how you managed the situation, and how you could have managed the situation differently. This also acts as a resource for others to learn how they may be able to manage similar situations.
+- Remain calm during your interactions, speak slowly, avoid arguing, avoid shouting, don’t use overly complicated jargon, and try to be understanding of others. Doing this will allow you to handle the situation in a calm manner and avoid conflict.
+- If you are imposing an unpopular ruling, explain clearly why you are doing it. If the competitor is young, it may be beneficial to involve their parent/guardian. Some competitors are easily embarrassed. One way to defuse that tension is to draw from personal experience, and let them know that the same situation has happened to you before.
+  - Example: a young competitor doesn’t start their timer correctly and becomes upset as their attempt has been given a DNF penalty. Speak to their parents, use more straightforward terms such as “disqualify” rather than DNF, to explain what happened. Show the competitor how to start the timer correctly. Tell them about a time it happened to you, maybe a time when you had a particularly good scramble. Give them tips on how you have avoided this happening again.
```

Comment:
Same thing here about the non-standard quotes character (“). All uses of it should be replaced with the standard Unicode double quote character (").